editor/debugger/editor_debugger_node.cpp

```diff
@@ -145,7 +146,13 @@ void EditorDebuggerNode::_text_editor_stack_goto(const ScriptEditorDebugger *p_d
 	if (file.is_empty()) {
 		return;
 	}
-	stack_script = ResourceLoader::load(file);
+	if (file.is_resource_file()) {
+		stack_script = ResourceLoader::load(file);
+	} else {
+		// If the script is built-in, it can be opened only if the scene is loaded in memory.
+		Ref<PackedScene> ps = ResourceLoader::load(file.get_slice("::", 0));
+		stack_script = ResourceLoader::load(file);
+	}
```

Comment:
Could maybe be simplified?
```suggestion
	if (!file.is_resource_file()) {
		// If the script is built-in, it can be opened only if the scene is loaded in memory.
		Ref<PackedScene> ps = ResourceLoader::load(file.get_slice("::", 0));
	}
	stack_script = ResourceLoader::load(file);
```

Reply:
Nevermind, after posting it I realized that `ps` then goes out of scope and bringing it to the outer scope is meh.